src/App.css

```diff
@@ -1,5 +1,4 @@
 .App {
-  text-align: center;
```

Comment:
There are some issues with text-alignment on other pages when you remove this I'd suggest instead just add the text-align: left css into the text that we need left aligned so that we don't need to find and fix the issues across all pages